src/acquisition/covidcast/csv_importer.py

```diff
@@ -379,9 +384,16 @@ def load_csv(filepath: str, details: PathDetails) -> Iterator[Optional[Covidcast
 
     try:
       table = pd.read_csv(filepath, dtype=CsvImporter.DTYPES)
-    except ValueError as e:
+    except pd.errors.DtypeWarning as e:
```

Comment:
whoah, nice catch!  does `read_csv` no longer raise the `ValueError`?  do we want to catch both?